libraries/AP_Scripting/tests/docs_check.py

```diff
@@ -0,0 +1,130 @@
+'''
+Reads two lua docs files and checks for differences
+
+python ./libraries/AP_Scripting/tests/docs_check.py "./libraries/AP_Scripting/docs/docs expected.lua" "./libraries/AP_Scripting/docs/docs.lua"
+'''
```

Comment:
```suggestion

AP_FLAKE8_CLEAN
'''
```

Reply:
I bet its not, I guess we'll find out....